### PR TITLE
Updated roslyn version to consume the latest nightly, and updated code

### PR DESCRIPTION
**Pull request by jinujoseph**
This is follow-up from https://github.com/dotnet/roslyn-analyzers/pull/1177 
Review the last commit 

Tagging @dotnet/analyzer-ioperation @genlu for review.

### CODE REVIEW

**Comment by mavasani** [comment]
Tests seem to be failing..

File: src/Analyzer.Utilities/Extensions/IEnumerableExtensions.cs [hunk 1]
```diff
@@ -71,7 +71,9 @@ public static IEnumerable<T> OrderBy<T>(this IEnumerable<T> source, Comparison<T
             return source.Where((Func<T, bool>)s_notNullTest);
         }
 
+#pragma warning disable CA1812
```
Comment (mavasani) [inline src/Analyzer.Utilities/Extensions/IEnumerableExtensions.cs]: I thought @333fred explicitly fixed this issue.

Comment (mavasani) [inline src/Analyzer.Utilities/Extensions/IEnumerableExtensions.cs]: Please file a bug to remove this suppression.

File: build/Targets/Versions.props [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <FakeSignVersion>0.9.2</FakeSignVersion>
+    <RoslynVersion>2.3.0-beta2-61719-05</RoslynVersion>
+    <RoslynAnalyzersVersion>1.3.0-beta1</RoslynAnalyzersVersion>
```
Comment (333fred) [inline build/Targets/Versions.props]: @jinujoseph, what version is this supposed to be? We don't have a 1.3.0-beta1 version, there are a bunch of warnings during package restore about getting 2.0.0-beta1.

**Pull request merged**